cmake_templates/qgsconfig.h.in

```diff
@@ -104,5 +104,7 @@
 
 #cmakedefine ENABLE_TESTS
 
+#cmakedefine WITH_QT5_KDE_FORK
```

Comment:
it would be nice if this could be determined from the Qt build.

Reply:
Yeah it can't -- I've checked with kdab and there's no way to do that (short of building a test application, but even then there's no easy way test we can implement to detect the fork)

Reply:
https://invent.kde.org/qt/qt/qtbase/-/merge_requests/75
let's see

Reply:
Ok! We'll still need this commit in the meantime though, at least until that request is merged and the different channels re-update from the fork...

Reply:
@3nids actually I'm wondering if we need to be even coarser then this and have an individual cmake switch to reflect each fix which is present in the actual version of the KDE fork being built off. There's no version numbering for the fork, yet we'll need some way to distinguish a qt build of the fork from say 2 months ago vs today's version of the fork..... :scream: 

So something like HAS_KDE_PDF_TRANSFORM_PATCH, HAS_KDE_WINDOW_SIZE_RESTORE_PATCH, ...

Reply:
I was wondering this too.
I was thinking that we are expecting people to keep up to date (if you pull QGIS, then you also fetch the last changes from KDE). I have no strong opinion here, I am just wondering if it's worth the headache. 

I could also try to bring the date in the variable defined in the PR. We could then assume `Qt build date = last fetch from Qt source`.

Reply:
I'll go with individual switches for now... It's easy enough to change in future if we need 👍